app/util/sentry/utils.test.ts

```diff
@@ -126,4 +129,225 @@ describe('captureSentryFeedback', () => {
       expectedUserFeedback,
     );
   });
+  describe('maskObject', () => {
+    it('mask our fixture of intial root state', () => {
```

Comment:
S: Could you test a state with actual values that would cover the specific cases you test in the util function? See how "accounts" for instance masks with a real value. I feel the fixture root state is too empty to provide an accurate masking test. You have to at least test it of course, but adding a more realistic one would help.

Also, just some naming:
```suggestion
    it('masks initial root state fixture', () => {
```

Reply:
Refactored on this commit: https://github.com/MetaMask/metamask-mobile/pull/10241/commits/26ba148811d8c108dfe5ebb850e4a5fe17d9a232